src/dp_abstraction/aca_dataplane_ovs.cpp

```diff
@@ -354,121 +361,138 @@ int ACA_Dataplane_OVS::update_neighbor_state_workitem(NeighborState current_Neig
 
   switch (current_NeighborState.operation_type()) {
   case OperationType::CREATE:
+    [[fallthrough]];
+  case OperationType::UPDATE:
+    [[fallthrough]];
+  case OperationType::INFO:
 
-    // TODO: add support for more than one fixed_ips in the future
-    assert(current_NeighborConfiguration.fixed_ips_size() == 1);
+    if (current_NeighborConfiguration.fixed_ips_size() <= 0) {
+      ACA_LOG_ERROR("NeighborConfiguration.fixed_ips_size: %d.\n",
+                    current_NeighborConfiguration.fixed_ips_size());
+      throw std::invalid_argument("NeighborConfiguration.fixed_ips_size is less than zero");
```

Comment:
This validation is in use more than one times. Let us keep the logic in a method if possible.

Reply:
fixed